source/fuzz/fuzzer.h

```diff
@@ -133,52 +178,41 @@ class Fuzzer {
 
   // Message consumer that will be invoked once for each message communicated
   // from the library.
-  MessageConsumer consumer_;
+  const MessageConsumer consumer_;
 
   // The initial binary to which fuzzing should be applied.
-  const std::vector<uint32_t>& binary_in_;
+  const std::vector<uint32_t> binary_in_;
 
   // Initial facts known to hold in advance of applying any transformations.
-  const protobufs::FactSequence& initial_facts_;
+  const protobufs::FactSequence initial_facts_;
 
   // A source of modules whose contents can be donated into the module being
   // fuzzed.
-  const std::vector<fuzzerutil::ModuleSupplier>& donor_suppliers_;
+  const std::vector<fuzzerutil::ModuleSupplier> donor_suppliers_;
```

Comment:
These are only used once during the initialization of the `Fuzzer::State`. It seems a waste of space to keep these around. Instead, we could do all the initialization in the constructor but then we would need to get rid of `kFailedToCreateSpirvToolsInterface` and `kInitialBinaryInvalid` and replace them with assertions. @afd, I'm happy to refactor all this if you are happy with this approach.

Reply:
These fields are removed in 2b452b4.